### DISCUSSION TITLE
Looking up package in yarn

### DISCUSSION THREAD

**Discussion by Cheprer** [uNetworking/uWebSockets.js#752] [Q&A] [answered]
Tried to search for yarn package (https://yarnpkg.com) mentioned in `Easily installed` in Readme section. Can't find it using the `uWebSockets.js@uNetworking/uWebSockets.js`. Can you please point me out how to find it? Or explain more about the fetching mechanism it is using and from where. I wanted to check downloads since downloads badge in Readme is static image.

**Comment by e3dio** [comment, marked as answer]
With NPM and Yarn you can install packages direct from GitHub with `npm i <username>/<repo>` and skip the NPM repo. I do that with my package https://github.com/e3dio/packBytes because publishing to NPM is a hassle so I just use Github. So where you are looking is not there, but Github tracks number of clones so you can see downloads etc 

![image](https://user-images.githubusercontent.com/85405955/173119756-dd7c9a98-269c-48df-b2a7-471a5ecd9cbe.png)

**Comment by ghost** [comment]
> I wanted to check downloads since downloads badge in Readme is static image.

GitHub only tracks 2 weeks a time so numbers are rough estimates, but they are probably in the lower end rather than the higher.

Ideally I would have my own server tracking downloads and also separating binaries on a platform basis (serving only Linux binaries to clients on Linux).

That's one feature NPM has rejected and ignored without any feedback. (NPM doesn't give a shit about our feedback).

**Comment by Cheprer** [comment]
Thank you guys for bringing more light into this. Cheers
